aws/v11.6.0/release.diff

```diff
@@ -0,0 +1,66 @@
+apiVersion: release.giantswarm.io/v1alpha1                         apiVersion: release.giantswarm.io/v1alpha1
+kind: Release                                                      kind: Release
+metadata:                                                          metadata:
+  annotations:                                                       annotations:
+    giantswarm.io/docs: https://docs.giantswarm.io/reference/cp        giantswarm.io/docs: https://docs.giantswarm.io/reference/cp
+  creationTimestamp: null                                            creationTimestamp: null
+  name: v11.5.6                                                 |    name: v11.6.0
+spec:                                                              spec:
+  apps:                                                              apps:
+  - name: cert-exporter                                              - name: cert-exporter
+    version: 1.2.3                                                     version: 1.2.3
+  - componentVersion: 0.9.0                                          - componentVersion: 0.9.0
+    name: cert-manager                                                 name: cert-manager
+    version: 1.0.8                                                     version: 1.1.0
```

Comment:
Why isn't this line marked as being different when it's clear that the version numbers are different?